theme/themes/pastanaga/extras/toolbar.less

```diff
@@ -19,6 +19,70 @@
   }
 }
 
+body:not(.has-sidebar):not(.has-sidebar-collapsed) {
```

Comment:
@nzambello @pnicolli sorry for having to ask again. This selector affect the standard theme, right? Could you elaborate again please what's the purpose of this?

Reply:
@nzambello @pnicolli ping? I'd like to merge and release this. :)

Reply:
This was a fix to enforce that and avoid a conflict what would be present with the `cms-ui` wrapper.
It doesn't affect the base theme itself but enforces this styles